### PR TITLE
Dev 1854 implement open source verificator design

### PR DESCRIPTION
**Pull request by martinpam**

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -66,10 +73,128 @@ document.addEventListener('DOMContentLoaded', () => {
...
+      files[i].name +
+      "</div>   </div> <div id=\"fileRightSegment\">" + 
+      ($isVerified ?
+        ""://"<div id=\"viewButton\"> View</div>" : 
+        !$isPdf ? 
+          "<div id=\"noView\"><span>Invalid file format</span></div>" : 
+            
+          "<div id=\"noView\"><span>"+ $errorMessage +"</span></div>")+//"<div id=\"viewButton\"> View</div>") +
+      "</div></div>"
+      $fileList.innerHTML += $html;
```
Comment (paul-em) [inline app/index.js:188]: This is a potential security risk. using innerHtml can lead to xss attacks. Please sanitize the user defined output in line 179 with this function:

```
function sanitizeHTML(text) {
  const element = document.createElement('div');
  element.innerText = text;
  return element.innerHTML;
}
```

File: app/verify.js [hunk 2]
```diff
@@ -129,7 +129,7 @@ export function verifyCumulatedHash(data) {
       return entryHash;
     })
     .join('');
-  return hashRaw(cumulatedHash);
+  return hashRaw(cumulatedHash);
```
Comment (paul-em) [inline app/verify.js:132]: Remove the whitespace please :) are you using any linter?

**Review by paul-em — CHANGES_REQUESTED** [review]

File: app/verify.js (hunk 2 above)
Comment (martinpam) [reply app/verify.js]: activated format on save now, so no more whitespaces from here on out :D (probably)

**Review by paul-em — CHANGES_REQUESTED** [review]
Oh no, your autoformat changed single quotes to double quotes 😱  you can edit this setting in your editor. We only use single quotes.

Also for future reference: If you do a format commit, do not add any other changes in there as it is impossible to find them and review them 😆

**Review by paul-em — APPROVED** [review]

**Pull request merged**